From e7537ca67b8660368015afb56b7c950c10afea0c Mon Sep 17 00:00:00 2001
Message-Id: <e7537ca67b8660368015afb56b7c950c10afea0c.1684964119.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1684964119.git.sharlatanus@gmail.com>
References: <cover.1684964119.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 15 May 2023 23:57:50 +0100
Subject: [PATCH v2 05/20] gnu: python-spherical-geometry: Add update note.

* gnu/packages/astronomy.scm (python-spherical-geometry): Package can't
be updated to the latest version due to an open issue in upstream where
system qb library is in use.
---
 gnu/packages/astronomy.scm | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 196aeba0e3..04a1895662 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1987,6 +1987,8 @@ (define-public python-astral
 (define-public python-spherical-geometry
   (package
     (name "python-spherical-geometry")
+    ;; XXX: Can't be updated to the latest see:
+    ;; https://github.com/spacetelescope/spherical_geometry/issues/227
     (version "1.2.22")
     (source
      (origin
-- 
2.39.2

